lib/resources/shim-head.js

```diff
@@ -2,6 +2,25 @@
 ((win) => {
   win.ELECTRON = true;
 
+  if (process.env.ELECTRON_PROTOCOL_SERVE_INDEX && window.location.protocol !== 'file:') {
```

Comment:
I suppose somebody could be using electron to load `index.html` over the network, but then they are pretty far outside `ember-electron`'s intended use case, so I'd like to understand what they are doing and why before writing code to try to anticipate and guess at it.